metricbeat/docs/fields.asciidoc

```diff
@@ -6728,7 +6728,7 @@ The current number of idle client connections waiting for a request.
 
 
 [[exported-fields-php_fpm]]
-== php_fpm Fields
+== PHP-FPM Fields
```

Comment:
@monicasarbu In my edits, I'm being consistent with our general approach of using the name of the product/service that's being monitored (for example, PHP-FPM) when we describe the module. Otherwise, especially in headings, it sort of looks like we are misspelling product names. But I'm wondering...do you think this might cause people to use the wrong module names in their configs (like php-fpm instead of php_fpm)? We do show a config example, so I might be overthinking this.